spec/tasks/send_pre_deadline_reminder_service_spec.rb

```diff
@@ -14,73 +14,19 @@
     end
   end
 
-  let!(:az_intake_with_email_notifications_and_df_import) {
-    create :state_file_az_intake,
-           df_data_imported_at: 2.minutes.ago,
-           email_address: 'test@example.com',
-           email_address_verified_at: 5.minutes.ago,
-           email_notification_opt_in: 1
-  }
-  let!(:az_intake_with_email_notifications_without_df_import) {
-    create :state_file_az_intake,
-           df_data_imported_at: nil,
-           email_address: 'test@example.com',
-           email_address_verified_at: 5.minutes.ago,
-           email_notification_opt_in: 1
-  }
-  let!(:nc_intake_with_text_notifications_and_df_import) {
-    create :state_file_az_intake,
-           df_data_imported_at: 2.minutes.ago,
-           phone_number: "+15551115511",
-           sms_notification_opt_in: 1,
-           phone_number_verified_at: 5.minutes.ago
-  }
-  let!(:nc_intake_with_unverified_text_notifications_and_df_import) {
-    create :state_file_nc_intake,
-           df_data_imported_at: 2.minutes.ago,
-           phone_number: "+15551115511",
-           sms_notification_opt_in: "yes",
-           email_address: 'test@example.com',
-           email_address_verified_at: 5.minutes.ago,
-           email_notification_opt_in: "no"
-  }
-  let!(:nc_intake_submitted) {
-    create :state_file_nc_intake,
-           df_data_imported_at: 2.minutes.ago,
-           email_address: 'test+01@example.com',
-           email_address_verified_at: 5.minutes.ago,
-           email_notification_opt_in: 1
-  }
-  let!(:efile_submission) { create :efile_submission, :for_state, data_source: nc_intake_submitted }
-  let!(:az_intake_has_received_reminder) {
-    create :state_file_az_intake, email_address: "test@example.com",
-           email_address_verified_at: 1.hour.ago,
-           email_notification_opt_in: 1,
-           df_data_imported_at: 2.minutes.ago,
-           message_tracker: { "messages.state_file.finish_return" => (Time.now - 2.hours).utc.to_s }
-  }
-  let!(:md_intake_disqualifying_df_data) {
-    create :state_file_md_intake,
-           df_data_imported_at: 2.minutes.ago,
-           email_address: 'test+01@example.com',
-           email_address_verified_at: 5.minutes.ago,
-           email_notification_opt_in: 1
-  }
-
```

Comment:
wondering if these fixtures were the test for the hashed_ssn/shared email scenarios

Reply:
this is for the pre-deadline reminder, but the post deadline reminder did not test for the hashed ssn. that was also confusing.

Reply:
o yeah that's not great